Python/PRP/Surface/pyCubicEnvironmap.cpp

```diff
@@ -0,0 +1,155 @@
+/* This file is part of HSPlasma.
+ *
+ * HSPlasma is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * HSPlasma is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with HSPlasma.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <PyPlasma.h>
+#include <PRP/Surface/plCubicEnvironmap.h>
+#include "pyCubicEnvironmap.h"
+#include "pyBitmap.h"
+
+extern "C" {
+
+static PyObject* pyCubicEnvironmap_new(PyTypeObject* type, PyObject* args, PyObject* kwds) {
+    pyCubicEnvironmap* self = (pyCubicEnvironmap*)type->tp_alloc(type, 0);
+    if (self != NULL) {
+        self->fThis = new plCubicEnvironmap();
+        self->fPyOwned = true;
+    }
+    return (PyObject*)self;
+}
+
+static PyObject* pyCubicEnvironmap_getLeftFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kLeftFace));
+}
+
+static PyObject* pyCubicEnvironmap_getRightFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kRightFace));
+}
+
+static PyObject* pyCubicEnvironmap_getFrontFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kFrontFace));
+}
+
+static PyObject* pyCubicEnvironmap_getBackFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kBackFace));
+}
+
+static PyObject* pyCubicEnvironmap_getTopFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kTopFace));
+}
+
+static PyObject* pyCubicEnvironmap_getBottomFace(pyCubicEnvironmap* self, void*) {
+    return pyMipmap_FromMipmap(self->fThis->getFace(plCubicEnvironmap::Faces::kBottomFace));
+}
+
+static PyMethodDef pyCubicEnvironmap_Methods[] = {
+    { NULL, NULL, 0, NULL }
```

Comment:
You can put `NULL` in the `tp_methods` field below instead of setting an empty list...